apps/ejabberd/src/ejabberd_s2s_in.erl

```diff
@@ -150,16 +149,17 @@ init([{SockMod, Socket}, Opts]) ->
                  {value, {_, S}} -> S;
                  _ -> none
              end,
-    {StartTLS, TLSRequired, TLSCertverify} = case ejabberd_config:get_local_option(s2s_use_starttls) of
-             UseTls when (UseTls==undefined) or (UseTls==false) ->
-                 {false, false, false};
-             UseTls when (UseTls==true) or (UseTls==optional) ->
-                 {true, false, false};
-             required ->
-                 {true, true, false};
-             required_trusted ->
-                 {true, true, true}
-         end,
+    {StartTLS, TLSRequired, TLSCertverify} =
+        case ejabberd_config:get_local_option(s2s_use_starttls) of
+            UseTls when (UseTls==undefined) or (UseTls==false) ->
+                {false, false, false};
+            UseTls when (UseTls==true) or (UseTls==optional) ->
+                {true, false, false};
+            required ->
+                {true, true, false};
+            required_trusted ->
+                {true, true, true}
+        end,
```

Comment:
I'd suggest extracting it into a pattern-matched function, like:
```
tls_params(undefined) -> {false, false, false};
tls_params(false) -> {false,false, false};
tls_params(true) -> {true, false, false};
...
```
IMO it would be much more readable.